src/sage/crypto/public_key/key_exchange/key_exchange_base.py

```diff
@@ -126,8 +128,9 @@ def alice_key_generate(self) -> tuple[Any, Any]:
         key exchange.
 
         OUTPUT:
-            A two tuple (secret_key, public_key) which is Alice's
-            secret and public keys
+
+        A two tuple (secret_key, public_key) which is Alice's
```

Comment:
```suggestion
        A 2-tuple (secret_key, public_key) which is Alice's
```
For consistency with the documentation of the other methods